### DISCUSSION TITLE
Problems after update 1.5.12

### DISCUSSION THREAD

**Discussion by robertnijhuis** [FrSkyRC/ETHOS-Feedback-Community#4368] [General]
Are there more people who experience problems after installing the last update (1.5. 12).
After the update I get the message "Invalid data" 
![error](https://github.com/user-attachments/assets/5e5b4b39-5e21-4663-8457-f05aea9fbfac)
at a certain point when I delete it and when you press OK it deletes the model. It is even so that I flew 3 hours with a glider today and when I wanted to fly again later I got the message and it deleted the model. I then did a total reset at home, reset everything and restored the backup of the models but then it keeps doing it. I then reset everything again and did a repair and then programmed a new model and that was also deleted.

**Comment by bsongis-frsky** [comment]
This has been fixed in 1.5.13, right?
